File: one.rb
<<<<<<< HEAD
#Birinci Odev
@me=[]
puts"How old are you?"
age=gets.chomp
@me.push(age)

puts "What is your name?"
name=gets.chomp
@me.push(name)

puts "What is your school name?"
school=gets.chomp
@me.push(school)

@sum = @me[1].length + @me[2].length

if @sum >7
	@me.push("bigger than seven")
else
	@me.clear
	@me.push("smaller than seven")
end

@me.each_with_index {|val, index| puts "[#{index}] #{val} " }
=======
@me = Array.new

age = gets.chomp
name = gets.chomp
school = gets.chomp
@me << age << name << school

@sum = (@me[1] + @me[2]).length
p @sum > 7 ? @me << "7'den büyük" : @me.clear << "7'den küçük"
>>>>>>> 3c0c209c4fffdea902ee4be6a36a23b3a7a6aadf

File: one.py
age = input("Yasiniz? ")
name = input("Adiniz? ")
school = input("Okulunuz? ")

me = [age,name,school]
sum = int(me[0])+ len(me[1]) + len(me[2])

if sum > 7:
	me.append("7'den büyük")
else:
	del me[:]
	me.append("7'den küçük")
print (me)